src/Aggregation/RawAggregation.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchDSL\Aggregation;
+
+use ONGR\ElasticsearchDSL\Aggregation\Type\BucketingTrait;
+
+/**
+ * Class representing a raw aggregation, that allows to set
+ * a custom type and body
+ */
+class RawAggregation extends AbstractAggregation
+{
+    use BucketingTrait;
```

Comment:
What if I want to use metric aggregation type with this raw? IMO there is a logic conflict for using it.